config/config.go

```diff
@@ -170,6 +170,12 @@ type SecurityConfig struct {
 	Certificates                     CertificatesConfig `json:"certificates"`
 }
 
+type NewRelicConfig struct {
+	Enabled    bool   `json:"enabled"`
```

Comment:
Do you think it make sense get rid of `Enabled` field, and instead just check if AppName is non empty?

Reply:
OK, can be done.